funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -203,8 +202,8 @@ def load_config(self):
         """
         logger.info("Loading endpoint local config")
 
-        working_dir = self.config.working_dir
-        if self.config.working_dir is None:
+        working_dir = self.config.log_dir
+        if self.config.log_dir is None:
             working_dir = "{}/{}".format(self.logdir, "worker_logs")
         logger.info("Setting working_dir: {}".format(working_dir))
```

Comment:
looks like this block is never used

Reply:
I should merge in changes from #439 to make sure that the log_dir is actually propagating to the executors.